Current state of the BEP (NOT FOR MERGING)

Lestropie opened the pull request:
This is a replacement of #6 as the BEP is being reinvested in.

The PR is not for merging, but to show the current state of the difference between the BEP and an up-to-date `master` of the BIDS specification.

Lestropie committed:
First commit for restructuring of DWI derivatives

Lestropie committed:
BEP016: Small fixes, and try to get links to headers working

Lestropie committed:
BEP016: More updates of internal header links

effigies committed:
ENH: Restore diffusion derivatives

Lestropie committed:
BEP016: Minor tweaks from first feedback round

- More clarity of distinction between requisite and optional files in output directory.
- Try using 3 spaces rather than 4 in non-code indentation; partly to try to get tables within dot point lists to render correctly, partly to improve editor software syntax highlighting.

Lestropie committed:
BEP016: Try to get links working within tables

Lestropie committed:
BEP016: More minor tweaking

- Various small re-wordings.
- Slightly more use of hyperlinks.
- Short introductions to "parameter terminology" and "data representations" sections.
- Be more explicit about normalised vs. non-normalised 3-vectors, so that structure more clusely mimics that of description of spherical coordinate representation.
- Rename hyperlink names to "parameter terminology" section to better separate from later "intrinsic / extrinsic model parameters" sections.

Lestropie committed:
BEP016: Transpose SH volume count tables

Lestropie committed:
BEP016: Provide example for parameter definitions

Lestropie committed:
BEP016: Change to single-file diffusion models

Based on suggestion in #5. If all model intrinsic parameters are incorporated into a single file, rather than dropping the "_parameter-<param>" field, instead enforce that parameter name "all" be used.

Lestropie committed:
BEP016: Initial explanation of filename

When introducing the file naming convention, give an example of the "_<model>" field.

Lestropie committed:
BEP016: File naming clarification

Re-arranged descriptions of intrinsic and extrinsic model parameters within the file naming section, and corrected a discordance in one dot point that was using an intrinsic model parameter filename path but discussing extrinsic model parameters.

Lestropie committed:
BEP016: Spelling fix

Lestropie committed:
BEP016: Fix missing anchor links

Lestropie committed:
BEP016: Move orientation specification

Provide information on specification of orientation data after the various models and model parameters have been explained.

oesteban committed:
MAINT: Split out tractography

This PR splits the tractography section from the diffusion derivatives
document, so that #5 is easier to merge.
The new ``05-diffusion-derivatives-tractography.md`` file will remain
orphaned, but kept there as a base for the time we tackle tractography.
It shouldn't be merged into the derivatives branch until it is ready.

arokem committed:
Merge pull request #9 from bids-standard/maint/split-tractography

[FIX] Split out tractography

oesteban committed:
Merge pull request #5 from bids-standard/dwi_derivatives_restructure

[ENH] Restructuring of DWI derivatives

effigies committed:
Merge remote-tracking branch 'upstream/master' into bep-016

Lestropie committed:
Merge remote-tracking branch 'bids-specification/master' into bep-016

Lestropie committed:
BEP016: Revise SH description

Revised based on MRtrix3/mrtrix3#1635.
Manual merge / cherry-pick of: c6539851, 8b278282, c6539851, ebbf0d3d, d7b4731d, 99e92e0a, 6bfc7849 due to unresolvable conflict in bids-standard/bids-bep016#8.

Lestropie committed:
BEP016: Initial conformity to markdown formatting

Lestropie committed:
BEP016: More formatting conformity changes

Lestropie committed:
BEP016: More formatting conformity changes

Lestropie committed:
BEP016: Attempted fixes to table cell padding

Lestropie committed:
BEP016: Minor fix to tractography derivatives for CI pass

Lestropie committed:
Merge pull request #30 from bids-standard/remark_fixes

[FIX] Formatting conformity to pass CI

Lestropie committed:
Merge branch 'bep-016' into sh_description_cherrypick

Conflicts:
	src/05-derivatives/05-diffusion-derivatives.md

Lestropie committed:
BEP016: Modify nomenclature around MRtrix3 SH coefficients

Lestropie committed:
BEP016: Revise preprocessed DWI data example

- Provide basic instructions rather than elaborating on rare use cases.
- Remove JSON dictionary on preprocessing steps utilised as these relate to provenance and are therefore out of scope.
Proposal for addressing bids-standard/bids-bep016#25.

Lestropie committed:
BEP016: Remove SH Descoteaux placeholder

Lestropie committed:
BEP016: Remove majority of models

Introduction and review of most models are intended to be deferred as per bids-specification/bids-bep016#39.

Lestropie committed:
BEP016: Formatting fix following removal of models

arokem committed:
Merge pull request #40 from bids-standard/remove_models

BEP016: Remove majority of models

francopestilli committed:
Merge pull request #37 from bids-standard/revise_preprocessed

[FIX] Revise preprocessed DWI data example

francopestilli committed:
Merge pull request #28 from bids-standard/sh_description_cherrypick

[ENH] SH description (take 2)

Lestropie committed:
BEP016: Strip out tractography content

arokem committed:
Fixes a typo caught by codespell

Lestropie committed:
Merge pull request #45 from bids-standard/typo-fix

FIX: typo caught by codespell

Lestropie committed:
BEP016: Initial draft of common filename suffix "model"

Lestropie committed:
BEP016: Rename "dti" to "tensor"

arokem committed:
Merge pull request #42 from bids-standard/strip_tractography

Strip out tractography content

Lestropie committed:
Merge pull request #47 from bids-standard/rename_tensor

BEP016: Rename "dti" to "tensor"

Lestropie committed:
Rename "_parameter-" entity to "_param-"

Lestropie committed:
DWI derivatives: Compulsory "parameter" entity

While the specification states that the "parameter" entity must always be defined---even if all model parameters are encoded within a single image, in which case value "all" must be used---two of the demonstrative examples did not obey such.

Lestropie committed:
Merge pull request #51 from bids-standard/param_entity

Rename "_parameter-" entity to "_param-"

Lestropie committed:
Merge remote-tracking branch 'origin/bep-016' into compulsory_parameter_entity

Lestropie committed:
DWI derivatives: Resolution between content changes

Resolves bids-standard/bids-bep016#52 (making "parameter" entity compulsory) against bids-standard/bids-bep016#51 (changing key for that parameter from "parameter" to "param").

Lestropie committed:
Merge remote-tracking branch 'origin/bep-016' into model_suffix

Conflicts:
	src/05-derivatives/05-diffusion-derivatives.md

Lestropie committed:
DWI derivatives: Fix tensor model example

Lestropie committed:
DWI derivatives: "model" as entity

francopestilli committed:
Merge pull request #46 from bids-standard/model_suffix

BEP016: Initial draft of common filename suffix "model"

Lestropie committed:
Merge branch 'bep-016' into compulsory_parameter_entity

Conflicts:
	src/05-derivatives/05-diffusion-derivatives.md

Lestropie committed:
Merge pull request #52 from bids-standard/compulsory_parameter_entity

DWI derivatives: Compulsory "parameter" entity

Lestropie committed:
Merge branch 'bep-016' into model_entity

Conflicts:
	src/05-derivatives/05-diffusion-derivatives.md

Lestropie committed:
Merge branch 'master' into bep-016

Lestropie committed:
Merge pull request #57 from bids-standard/model_entity

"model" as an entity

Lestropie committed:
DWI models: Restore renaming "dti" to "tensor"

Changes in #47 were lost in the process of conflict merging.

Lestropie committed:
Merge pull request #60 from bids-standard/restore_rename_tensor

DWI models: Restore renaming "dti" to "tensor"

Lestropie committed:
BEP016: Remove reference to old terminology

Formerly known as "intrinsic" model parameters are now (currently) referred to as simply "model parameters".

Lestropie committed:
Merge pull request #63 from bids-standard/revert_intrinsic

BEP016: Remove reference to old terminology

Lestropie committed:
BEP016: Revert "_param-all"

In cases where all model parameters can be encapsulated in a single NIfTI image, revert back to the case where the "param" entity is omitted from the file name.
Reverts https://github.com/bids-standard/bids-bep016/pull/52.

Lestropie committed:
Merge pull request #64 from bids-standard/no_paramall

Revert "_param-all"

Lestropie committed:
BEP016: Adopt "model fit parameters" terminology

In order to better disambiguate the various types of "parameters" defined, change "model parameters" to "model fit parameters".

Lestropie committed:
BEP016: Remove references to removed "pdf" data representation

Lestropie committed:
Merge pull request #65 from bids-standard/model_fit_parameters

BEP016: Adopt "model fit parameters" terminology

Lestropie committed:
Merge pull request #66 from bids-standard/remove_pdf

BEP016: Remove references to removed "pdf" data representation

Lestropie committed:
DWI derivatives: Make tensor a data representation

Lestropie committed:
DWI derivatives: Fix ModelDescription in exemplars

Specification requests field "ModelDescription", but exemplars at end of document instead erroneously used key "Model".

Lestropie committed:
Merge pull request #75 from bids-standard/model_description

DWI derivatives: Fix ModelDescription in exemplars

PeerHerholz committed:
update based on current BIDS spec master

This commit/PR updates the bids-BEP016 repository to incorporate the changes/current state in the bids-specification repo.

arokem committed:
Merge pull request #78 from PeerHerholz/update_specification

synch bids-bep016 with current state of bids-specification [MISC]

arokem commented:
Looks like there are still a lot of unrelated things even after merging #78 in here.

arokem committed:
Merge branch 'master' into bep-016

arokem commented:
Ok - now merged with the specification `master` branch, this is (for now) the actual diff against the specification.

arokem committed:
Merge branch 'master' into bep-016

arokem commented on src/derivatives/05-diffusion-derivatives.md:
```diff
@@ -0,0 +1,604 @@
+<pipeline_name>/
+    sub-<participant_label>/
+        dwi/
+            <source_keywords>[_space-<space>]_model-<label>_param-<param1>_model.nii[.gz]
```
Should we change this suffix to `_mfp`, along the lines of the [ongoing work on dimensionality reduction networks](https://docs.google.com/document/d/1GTWsj0MFQedXjOaNk6H0or6IDVFyMAysrJ9I4Zmpz2E/edit#bookmark=id.24bbnfrlz01u)?

francopestilli replied:
I think this change was agreed on the dimensionality reduction BEP after we last discussed with @lestropie so Rob did know. Let's change it here too. Happy to get started when I back or Peer perhaps can?

Lestropie replied:
I've not followed discussions elsewhere RE file suffices; too overloaded. If MFP / MDP may catch on more broadly, can crossref to #69 / #68 / #50 and start a dedicated PR.

PeerHerholz committed:
Merge branch 'master' into bep-016

arokem commented on src/derivatives/05-diffusion-derivatives.md:
@arokem needs to write this for the Descoteaux SH basis as well

arokem committed:
Adds the descoteaux 2007 convention.

Based on https://github.com/dipy/dipy/blob/master/dipy/reconst/shm.py#L427-L430

arokem committed:
Merge pull request #80 from arokem/adds_descoteaux

Adds the descoteaux 2007 convention.

arokem committed:
Change from strange html encoding to "<" and ">"

arokem committed:
Merge pull request #81 from arokem/adds_descoteaux

Change from strange html encoding to "<" and ">"

arokem committed:
Fix codespell-detected British spellings.

arokem committed:
Merge pull request #82 from arokem/fix-british-spellings

Fix codespell-detected British spellings.

arokem committed:
Merge branch 'master' into bep-016

arokem commented:
Updated with current master branch on the upstream fork of the spec.

arokem committed:
Replace the latin "etc" with ellipses.

arokem committed:
Merge pull request #83 from arokem/fix-validation

Replace the latin "etc" with ellipses.

dependabot[bot] committed:
Bump actions/checkout from 3 to 4

Bumps [actions/checkout](https://github.com/actions/checkout) from 3 to 4.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v3...v4)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

arokem committed:
Merge branch 'master' into bep-016

Lestropie committed:
Merge branch 'bep-016' into tensor_as_representation

Lestropie committed:
Merge pull request #74 from bids-standard/tensor_as_representation

DWI derivatives: Make tensor a data representation

Lestropie committed:
BEP016: Major changes

- Reject suffices "mfp" (or earlier "_model") / "mfp" in favour of "_dwimap". Instead adopt distinction of "model metadata" vs. "parameter metadata".
- Reject use of advanced inheritance in favour of listing all relevant metadata for each data file in the sidecar JSON.
- Greater use of sub-dictionaries in JSON files to assist in separating metadata relevant to a model as a whole vs. only that particular parameter of the model.

Lestropie committed:
BEP016: Fis JSON formatting in demonstrative examples

Lestropie committed:
Diffusion models: Forbid negative spherical coordinate radius

francopestilli reviewed (approved):
This looks all good to me @arokem shall we approve?

francopestilli commented:
@Lestropie @arokem after this is approved, can we please merge this into the main spec? @effigies

arokem commented:
I believe that next steps are: 

1. Review and eventually merge #92 into this PR. I am in the midst of a review of that one, which I hope to complete today. 
2. Move this branch into a PR against bids-standard/bids-specification, so that we can hash out some of the remaining questions (see https://github.com/bids-standard/bids-bep016/pull/92#issue-2264977772) in that broader format. 
3. After iterating there and coming to consensus, we would still need to complete all the work on the BEP, before it can be merged into the spec. The next two steps are: (a) Examples and (b)Schema. The BEP cannot be considered ready to merge before these steps are completed (see [this page](https://bids-extensions.readthedocs.io/en/latest/submission/) for details).

Maybe you were referring specifically to item 2 above when you said "merge into the main spec"?

francopestilli commented:
Ok. Thanks let me know if I can help with the review

arokem committed:
Diffusion model derivatives: Spelling fixes

Lestropie commented:
-   Before actually constructing a PR, I think we need to go through the full Issue list. Some items can be deferred for inclusion after the first version of the extension, that's fine, but we need to explicitly flag such, whereas anything that's a requisite decision about a feasible first version needs to be resolved. I expect the maintainers / steering committee wouldn't like receiving an extension proposal where there's a lot of ambiguous outstanding Issues.

-   For the examples, the [existing data](https://github.com/bids-standard/bids-examples) appear to be exclusively raw data (ie. no derivatives), and the [README file explicitly states](https://github.com/bids-standard/bids-examples/blob/1.9.0/README.md?plain=1#L12) that it corresponds to raw data. The latter is maybe slightly ambiguous as they could be using "raw" to refer to data files in comparison to metadata files, as opposed to raw vs. derived. I'm interested to know if the "common derivatives" extension included providing example data. In the absence of such, we'd need to discuss with maintainers / steering committee about whether example derivatives should be going into the same examples repository vs. a different repository.

francopestilli commented:
Hi @Lestropie it is my understanding that the full issue list cannot be addressed as it goes beyond the scope of the PR as it requires changes to BIDS. I proposed constructing the PR and discussing what is needed given what BIDSbis now and discussing the other issues in the context of BIDS 2.0. we are 4 years in in some major issues without working solutions. I think it is fair to ask to wait for BIDS 2.0 for your major requests.

arokem committed:
Merge pull request #92 from bids-standard/dwimap_and_no_inheritance

Major refactoring for upstream changes / blocks

arokem committed:
Merge branch 'master' into bep-016

arokem commented:
I just merged the upstream `master` branch into this one, so this is in sync with bids-standard/bids-specification

arokem committed:
Apply US spelling. Addresses current codespell CI failure.

arokem committed:
Markdown table linting

arokem committed:
Get rid of a few latin phrases (i.e., "e.g.").

Lestropie committed:
DWI models: Define "bvec" orientation reference

Lestropie committed:
DWI models: Force presence of "param" entity

Lestropie committed:
DWI models: add "ParameterURL" metadata field

Lestropie committed:
Merge remote-tracking branch 'origin/bep-016' into dependabot/github_actions/actions/checkout-4

Conflicts:
	.github/workflows/schemacode_ci.yml
	.github/workflows/validation.yml

Lestropie committed:
Clarify metadata fields relevance (#102)

* DWI models: Clarify metadata fields relevance

* DWI models: Typo fix

---------

Co-authored-by: Ariel Rokem <arokem@gmail.com>

Lestropie committed:
Merge pull request #87 from bids-standard/dependabot/github_actions/actions/checkout-4

Bump actions/checkout from 3 to 4

Lestropie committed:
Merge pull request #97 from arokem/us-spelling

Apply US spelling. Addresses current codespell CI failure.

arokem committed:
Merge branch 'bep-016' into metadata_parameterurl

arokem committed:
Merge pull request #101 from bids-standard/metadata_parameterurl

Add "ParameterURL" metadata field

arokem committed:
Merge branch 'bep-016' into force_param_entity

arokem committed:
Merge pull request #99 from bids-standard/force_param_entity

DWI models: Force presence of "param" entity

arokem committed:
Removes one more i.e.

Lestropie committed:
Merge pull request #103 from arokem/one_more_latin

Removes one more i.e.

Lestropie committed:
New appendix page: spherical harmonics

Lestropie committed:
BEP016: Utilize filesystem macro

Lestropie committed:
BEP016: Reformat demonstrative examples

Lestropie committed:
BEP016: Don't use filesystem macros for templates

Lestropie committed:
BEP016: Comment filesystem macros

Lestropie committed:
Merge pull request #105 from bids-standard/filesystem_macros

Filesystem macros

Lestropie committed:
Appendix: SH: Fix LaTeX equations

Co-authored-by: Ariel Rokem <arokem@gmail.com>

Lestropie committed:
DWI models: Update bvec reference

- FSL bedpostx command fibre orientations stored as spherical coordinates are confirmed to use the "bvec" reference frame.
- Enforcing azimuth angle to obey the right hand rule about the zenith axis would necessitate direct modification of image intensities from bedpostx outputs to store. Therefore it would be preferable to instead define the sign of the azimuth angle based on the direction of the second reference axis.
Closes #95.
Closes #94.

arokem committed:
Merge branch 'master' into bep-016

Lestropie committed:
Merge remote-tracking branch 'origin/bep-016' into bvec_reference_axes

Conflicts:
	src/derivatives/05-diffusion-derivatives.md

Lestropie committed:
Merge pull request #98 from bids-standard/bvec_reference_axes

DWI models: Define "bvec" orientation reference

arokem commented:
IIUC, we need to resolve this discussion: https://github.com/bids-standard/bids-bep016/pull/104/files#r1605423106, and after that we can convert this to a PR against bids-standard/bids-specification:master. Is that others' understanding as well? Do folks want to weigh in on that terminology discussion? @francopestilli : do you have any thoughts on nomenclature for l/lmax?

Lestropie commented:
A PR without #71 and with no exemplar data generated via #23 would IMO be laughed off. You can't demand that the community format their data to your decree whilst literally never having done it yourself. Please give me time to generate a BIDS App to generate and save the derivatives currently exemplified in the specification document, so that they can be moved from there to `bids-examples`. I had to deal with the [reference frame handling issues](https://github.com/Lestropie/DWI_metadata/tree/50df2ce1c1c92a01d4eae7c5b0c6f94a0a0f9446) first.

PeerHerholz commented:
Hi @Lestropie,

concerning this, I started working on a respective `BIDS App` a while ago: [bids_bep16_conv](https://github.com/PeerHerholz/bids_bep16_conv). It's a fully working `BIDS App` and we would only need to update the output formatting to conform to the current up-to-date version. Thus, if you would consider this feasible instead of starting a new `BIDS App`, I'm happy to help! No worries of course if not, e.g. you would prefer a different implementation, etc. .

Cheers, Peer

Lestropie commented:
My comment was on the premise that the converter was being restricted in scope to exclusively conversion of pre-generated model fitting outputs, in which case another tool would be required to run the fitting then the conversion. Been a long break between bursts so certain details will still be a bit fuzzy. If in its current state that tool does both fitting and conversion, then that's the obvious starting point; but as per https://github.com/PeerHerholz/bids_bep16_conv/issues/7 longer term we'll want to think about separation of those two steps.

arokem committed:
Merge branch 'master' into bep-016

dependabot[bot] committed:
chore(deps): bump codecov/codecov-action from 4 to 5

Bumps [codecov/codecov-action](https://github.com/codecov/codecov-action) from 4 to 5.
- [Release notes](https://github.com/codecov/codecov-action/releases)
- [Changelog](https://github.com/codecov/codecov-action/blob/main/CHANGELOG.md)
- [Commits](https://github.com/codecov/codecov-action/compare/v4...v5)

---
updated-dependencies:
- dependency-name: codecov/codecov-action
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

arokem committed:
Merge branch 'master' into bep-016

dependabot[bot] committed:
chore(deps): bump astral-sh/setup-uv from 6 to 7

Bumps [astral-sh/setup-uv](https://github.com/astral-sh/setup-uv) from 6 to 7.
- [Release notes](https://github.com/astral-sh/setup-uv/releases)
- [Commits](https://github.com/astral-sh/setup-uv/compare/v6...v7)

---
updated-dependencies:
- dependency-name: astral-sh/setup-uv
  dependency-version: '7'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump the actions-infrastructure group across 1 directory with 5 updates

Bumps the actions-infrastructure group with 5 updates in the / directory:

| Package | From | To |
| --- | --- | --- |
| [actions/checkout](https://github.com/actions/checkout) | `5` | `6` |
| [actions/upload-artifact](https://github.com/actions/upload-artifact) | `4` | `5` |
| [actions/download-artifact](https://github.com/actions/download-artifact) | `5` | `6` |
| [actions/setup-node](https://github.com/actions/setup-node) | `4` | `6` |
| [actions/setup-python](https://github.com/actions/setup-python) | `5` | `6` |



Updates `actions/checkout` from 5 to 6
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v5...v6)

Updates `actions/upload-artifact` from 4 to 5
- [Release notes](https://github.com/actions/upload-artifact/releases)
- [Commits](https://github.com/actions/upload-artifact/compare/v4...v5)

Updates `actions/download-artifact` from 5 to 6
- [Release notes](https://github.com/actions/download-artifact/releases)
- [Commits](https://github.com/actions/download-artifact/compare/v5...v6)

Updates `actions/setup-node` from 4 to 6
- [Release notes](https://github.com/actions/setup-node/releases)
- [Commits](https://github.com/actions/setup-node/compare/v4...v6)

Updates `actions/setup-python` from 5 to 6
- [Release notes](https://github.com/actions/setup-python/releases)
- [Commits](https://github.com/actions/setup-python/compare/v5...v6)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-version: '6'
  dependency-type: direct:production
  update-type: version-update:semver-major
  dependency-group: actions-infrastructure
- dependency-name: actions/upload-artifact
  dependency-version: '5'
  dependency-type: direct:production
  update-type: version-update:semver-major
  dependency-group: actions-infrastructure
- dependency-name: actions/download-artifact
  dependency-version: '6'
  dependency-type: direct:production
  update-type: version-update:semver-major
  dependency-group: actions-infrastructure
- dependency-name: actions/setup-node
  dependency-version: '6'
  dependency-type: direct:production
  update-type: version-update:semver-major
  dependency-group: actions-infrastructure
- dependency-name: actions/setup-python
  dependency-version: '6'
  dependency-type: direct:production
  update-type: version-update:semver-major
  dependency-group: actions-infrastructure
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump peter-evans/create-pull-request from 7 to 8

Bumps [peter-evans/create-pull-request](https://github.com/peter-evans/create-pull-request) from 7 to 8.
- [Release notes](https://github.com/peter-evans/create-pull-request/releases)
- [Commits](https://github.com/peter-evans/create-pull-request/compare/v7...v8)

---
updated-dependencies:
- dependency-name: peter-evans/create-pull-request
  dependency-version: '8'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Lestropie committed:
Merge pull request #109 from bids-standard/dependabot/github_actions/codecov/codecov-action-5

chore(deps): bump codecov/codecov-action from 4 to 5

Lestropie committed:
Merge pull request #111 from bids-standard/dependabot/github_actions/astral-sh/setup-uv-7

chore(deps): bump astral-sh/setup-uv from 6 to 7

Lestropie committed:
Merge pull request #114 from bids-standard/dependabot/github_actions/peter-evans/create-pull-request-8

chore(deps): bump peter-evans/create-pull-request from 7 to 8

Lestropie committed:
Merge branch 'bep-016' into dependabot/github_actions/actions-infrastructure-f2d4b981e8

Lestropie committed:
Merge pull request #113 from bids-standard/dependabot/github_actions/actions-infrastructure-f2d4b981e8

chore(deps): bump the actions-infrastructure group across 1 directory with 5 updates

Lestropie committed:
Merge branch 'bep-016' into sh_to_appendix

Conflicts:
	src/derivatives/05-diffusion-derivatives.md

Lestropie committed:
Appendix: SH: Remove use of SH "degree"

Use of "order" has been adopted across the documentation.

Lestropie committed:
BEP016: Renaming relevant to FSL

- Rename exemplar data and metadata files encoding the FSL bedpostx ball-and-sticks model from "_model-bs" to "_model-ballsticks" to avoid negative acronym connotation.
- Rename permissble value for "Reference" metadata from "bvec" to "fsl": this axis reference convention is not specific to the "bvec" file format but is a convention utilised throughout the FSL software.

Lestropie committed:
Merge pull request #104 from bids-standard/sh_to_appendix

New appendix page: spherical harmonics

Lestropie committed:
Merge remote-tracking branch 'origin/bep-016' into fsl_renaming

Conflicts:
	src/derivatives/05-diffusion-derivatives.md

Lestropie committed:
Merge pull request #115 from bids-standard/fsl_renaming

BEP016: Renaming relevant to FSL

dependabot[bot] committed:
chore(deps): bump the actions-infrastructure group with 2 updates

Bumps the actions-infrastructure group with 2 updates: [actions/upload-artifact](https://github.com/actions/upload-artifact) and [actions/download-artifact](https://github.com/actions/download-artifact).


Updates `actions/upload-artifact` from 5 to 7
- [Release notes](https://github.com/actions/upload-artifact/releases)
- [Commits](https://github.com/actions/upload-artifact/compare/v5...v7)

Updates `actions/download-artifact` from 6 to 8
- [Release notes](https://github.com/actions/download-artifact/releases)
- [Commits](https://github.com/actions/download-artifact/compare/v6...v8)

---
updated-dependencies:
- dependency-name: actions/upload-artifact
  dependency-version: '7'
  dependency-type: direct:production
  update-type: version-update:semver-major
  dependency-group: actions-infrastructure
- dependency-name: actions/download-artifact
  dependency-version: '8'
  dependency-type: direct:production
  update-type: version-update:semver-major
  dependency-group: actions-infrastructure
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump codecov/codecov-action from 5 to 6

Bumps [codecov/codecov-action](https://github.com/codecov/codecov-action) from 5 to 6.
- [Release notes](https://github.com/codecov/codecov-action/releases)
- [Changelog](https://github.com/codecov/codecov-action/blob/main/CHANGELOG.md)
- [Commits](https://github.com/codecov/codecov-action/compare/v5...v6)

---
updated-dependencies:
- dependency-name: codecov/codecov-action
  dependency-version: '6'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump tabulate from 0.9.0 to 0.10.0

Bumps [tabulate](https://github.com/astanin/python-tabulate) from 0.9.0 to 0.10.0.
- [Changelog](https://github.com/astanin/python-tabulate/blob/master/CHANGELOG)
- [Commits](https://github.com/astanin/python-tabulate/compare/v0.9.0...v0.10.0)

---
updated-dependencies:
- dependency-name: tabulate
  dependency-version: 0.10.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump pymdown-extensions from 10.15 to 10.21.3

Bumps [pymdown-extensions](https://github.com/facelessuser/pymdown-extensions) from 10.15 to 10.21.3.
- [Release notes](https://github.com/facelessuser/pymdown-extensions/releases)
- [Commits](https://github.com/facelessuser/pymdown-extensions/compare/10.15...10.21.3)

---
updated-dependencies:
- dependency-name: pymdown-extensions
  dependency-version: 10.21.3
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump myst-parser from 4.0.1 to 5.1.0

Bumps [myst-parser](https://github.com/executablebooks/MyST-Parser) from 4.0.1 to 5.1.0.
- [Release notes](https://github.com/executablebooks/MyST-Parser/releases)
- [Changelog](https://github.com/executablebooks/MyST-Parser/blob/master/CHANGELOG.md)
- [Commits](https://github.com/executablebooks/MyST-Parser/compare/v4.0.1...v5.1.0)

---
updated-dependencies:
- dependency-name: myst-parser
  dependency-version: 5.1.0
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump sphinx from 8.2.3 to 9.0.4

Bumps [sphinx](https://github.com/sphinx-doc/sphinx) from 8.2.3 to 9.0.4.
- [Release notes](https://github.com/sphinx-doc/sphinx/releases)
- [Changelog](https://github.com/sphinx-doc/sphinx/blob/v9.0.4/CHANGES.rst)
- [Commits](https://github.com/sphinx-doc/sphinx/compare/v8.2.3...v9.0.4)

---
updated-dependencies:
- dependency-name: sphinx
  dependency-version: 9.0.4
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump myst-nb from 1.2.0 to 1.4.0

Bumps [myst-nb](https://github.com/executablebooks/myst-nb) from 1.2.0 to 1.4.0.
- [Release notes](https://github.com/executablebooks/myst-nb/releases)
- [Changelog](https://github.com/executablebooks/MyST-NB/blob/main/CHANGELOG.md)
- [Commits](https://github.com/executablebooks/myst-nb/compare/v1.2.0...v1.4.0)

---
updated-dependencies:
- dependency-name: myst-nb
  dependency-version: 1.4.0
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump tornado from 6.5.1 to 6.5.6

Bumps [tornado](https://github.com/tornadoweb/tornado) from 6.5.1 to 6.5.6.
- [Changelog](https://github.com/tornadoweb/tornado/blob/master/docs/releases.rst)
- [Commits](https://github.com/tornadoweb/tornado/compare/v6.5.1...v6.5.6)

---
updated-dependencies:
- dependency-name: tornado
  dependency-version: 6.5.6
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Lestropie committed:
Merge pull request #126 from bids-standard/dependabot/uv/tornado-6.5.6

chore(deps): bump tornado from 6.5.1 to 6.5.6

Lestropie committed:
Merge pull request #119 from bids-standard/dependabot/github_actions/actions-infrastructure-985357984d

chore(deps): bump the actions-infrastructure group with 2 updates

Lestropie committed:
Merge pull request #120 from bids-standard/dependabot/github_actions/codecov/codecov-action-6

chore(deps): bump codecov/codecov-action from 5 to 6

Lestropie committed:
Merge pull request #121 from bids-standard/dependabot/uv/tabulate-0.10.0

chore(deps): bump tabulate from 0.9.0 to 0.10.0

dependabot[bot] committed:
chore(deps): bump tornado from 6.5.6 to 6.5.7

Bumps [tornado](https://github.com/tornadoweb/tornado) from 6.5.6 to 6.5.7.
- [Changelog](https://github.com/tornadoweb/tornado/blob/master/docs/releases.rst)
- [Commits](https://github.com/tornadoweb/tornado/compare/v6.5.6...v6.5.7)

---
updated-dependencies:
- dependency-name: tornado
  dependency-version: 6.5.7
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump aiohttp from 3.12.13 to 3.14.1

---
updated-dependencies:
- dependency-name: aiohttp
  dependency-version: 3.14.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Lestropie committed:
Merge pull request #122 from bids-standard/dependabot/uv/pymdown-extensions-10.21.3

chore(deps): bump pymdown-extensions from 10.15 to 10.21.3

Lestropie committed:
Merge pull request #123 from bids-standard/dependabot/uv/myst-parser-5.1.0

chore(deps-dev): bump myst-parser from 4.0.1 to 5.1.0

Lestropie committed:
Merge pull request #124 from bids-standard/dependabot/uv/sphinx-9.0.4

chore(deps-dev): bump sphinx from 8.2.3 to 9.0.4

Lestropie committed:
Merge pull request #125 from bids-standard/dependabot/uv/myst-nb-1.4.0

chore(deps-dev): bump myst-nb from 1.2.0 to 1.4.0

Lestropie committed:
Merge pull request #127 from bids-standard/dependabot/uv/tornado-6.5.7

chore(deps): bump tornado from 6.5.6 to 6.5.7

Lestropie committed:
Merge pull request #128 from bids-standard/dependabot/uv/aiohttp-3.14.1

chore(deps): bump aiohttp from 3.12.13 to 3.14.1

julia-pfarr committed:
Add GitHub Action to sync community-meeting labeled issues/PRs to project board

Committed via https://github.com/asottile/all-repos

julia-pfarr committed:
Add GitHub Action to sync community-meeting labeled issues/PRs to project board

Committed via https://github.com/asottile/all-repos
